### ISSUE TITLE
[Zelda TOTK] Terrain going dark

### ISSUE THREAD

**Issue by IVirg0** [yuzu-emu/yuzu#10793]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Affected Build(s)

yuzu 1470

### Description of Issue

After sometimes the terrain is dark this can happen too when you pass time (sleep or wait at firecamp).
Video:
https://www.youtube.com/watch?v=W2NBqEatHxw
Setting:
![image](https://github.com/yuzu-emu/yuzu/assets/82561509/d4634002-50c3-4c20-ad02-96883c16e1ba)
![image1](https://github.com/yuzu-emu/yuzu/assets/82561509/d6593e9e-5e58-4984-8380-5087f48ed5f0)
![image2](https://github.com/yuzu-emu/yuzu/assets/82561509/e9e492e0-90cd-446a-ab5e-91789f9a3e18)

### Expected Behavior

Should not going dark after sometimes.
Thing i try to fix it.

Delete all my pipeline cache

Disable all my mods.

Disable These option:
-Disk Pipeline Cache
-Use Async GPU Emulation
-Accelerate ASTC Texture decoding
-ASTC Recompression Set on BC3 
-Force maximum clocks
-Decode ASTC textures Asynchronously (Hack)
-Enable Reactive Flushing
-Use Async Shader building (Hack)
-Use fast GPU Time (Hack)
-Use Vulkan Pipeline Cache
-Anisotropic Filtering set to auto and x16

### Reproduction Steps

Go on Lookout Landing camp sleep in the underground bed (Good weather is better to determine if the glitch is there).
After that teleport to the Sheika tower and he should be temporary fixed.

Game Version: 1.1.0
Save file:
[0100F2C0115B6000.zip](https://github.com/yuzu-emu/yuzu/files/11761652/0100F2C0115B6000.zip)
Mod Pack:
https://drive.google.com/file/d/1R7PYMiiqS3545_K_IixbwYA5_iwILnZV/view?usp=sharing

### Log File

Log File:
[yuzu_log.txt](https://github.com/yuzu-emu/yuzu/files/11761861/yuzu_log.txt)

### System Configuration

CPU: i9-12900K
GPU/Driver: RTX 3060 12gb / 536.23
RAM: 48GB ram 3200Mhz
The Game is on a Nvme
OS: Windows 10

**Comment by Squall-Leonhart** [comment]
Mod incompatibility, and Yuzu Fix is no longer required.

**Comment by IVirg0** [comment]
I have a NSP version of 1.1.0 with absolutely no mods running with it.
![image](https://github.com/yuzu-emu/yuzu/assets/82561509/e69bb142-19d2-49da-b5f4-cd816689db99)
I also removed the "Force Maximum clocks" option.
I tried versions 1.0.0, 1.1.1, and 1.1.2 (NSP format).
I installed EA 3685 and tried it with each version.
I still managed to reproduce this glitch.

PS:
I'm not really sure what a merged XCI is. Is it a game merged with an update into an XCI file?
